### DISCUSSION TITLE
Can a callback function be added to call after a tile request is successful or failed?

### DISCUSSION THREAD

**Discussion by longhaiyan-lhy** [maplibre/maplibre-gl-js#2943] [Q&A]
Whether a callback function can be added to the RequestManager class to return the URL and state after a tile request is successful or fails so that the downstream system can collect statistics on tile invoking.

**Comment by HarelM** [comment]
Anything is possible.
There are some events related to it. I would recommend better understanding how things work and read the docs more thoroughly.
After that feel free write a more elaborate issue.
